doc/performance-investigations/0012-increase-machine-type-core-count/summary.md

```diff
@@ -0,0 +1,45 @@
+# Investigate Machine Type Architecture Performance
+
+EQ Runner currently uses a machine type of `custom-4-4096` (4 vCPU cores, 4096MB of RAM). This investigation looked at the impact of using larger instances with more pods on the performance of the runner application.
+
+## Benchmark profile
+
+| Option                 | Value                        |
+| ---------------------- | ---------------------------- |
+| Requests file          | census_household_gb_eng.json |
+| Run time               | 20m                          |
+| User wait time minimum | 1                            |
+| User wait time maximum | 2                            |
+| Clients                | 256                          |
```

Comment:
It might be clear if that was `Clients per core` which was 16 just to make it clear each pod wasn't run with 256.

Reply:
good point 👍🏻